apriltag_ros/src/common_functions.cpp

```diff
@@ -435,19 +444,10 @@ void TagDetector::addImagePoints (
     apriltag_detection_t *detection,
     std::vector<cv::Point2d >& imagePoints) const
 {
-  // Add to image point vector the tag corners in the image frame
-  // Going counterclockwise starting from the bottom left corner
-  double tag_x[4] = {-1,1,1,-1};
-  double tag_y[4] = {1,1,-1,-1}; // Negated because AprilTag tag local
-                                 // frame has y-axis pointing DOWN
-                                 // while we use the tag local frame
-                                 // with y-axis pointing UP
-  for (int i=0; i<4; i++)
+  //population of intrinsic-free corner pixel coordinates 
+  for(int i=3;i!=-1;i--) //clockwise from top left
```

Comment:
Looking at the previous implementation, `imagePoints` was being populated {0, 1, 2, 3}.

Looking at the new changes, it is now {3, 2, 1, 0}.  This might have other downstream effects that break code that makes assumptions about this ordering, so please put it back to the original order.